benchmark/run_bench.sh

```diff
@@ -163,6 +163,7 @@ while [ $# -gt 0 ]; do
       echo "-req              req size in bytes, default value is 1"
       echo "-resp             resp size in bytes, default value is 1"
       echo "-rpc_type         valid values are unary|streaming, default is unary"
+      exit 1      
```

Comment:
It would be more natural to `exit 0` in this case, since there was no error.

Reply:
ok，I have updated the code